metadata-ingestion/src/datahub/ingestion/source/aws/glue.py

```diff
@@ -123,6 +123,10 @@ class GlueSourceConfig(AwsSourceConfig, PlatformSourceConfigBase, GlueProfilingC
         default=False,
         description="If an S3 Objects Tags should be created for the Tables ingested by Glue.",
     )
+    s3_tags_ignore_virtual_view_tables: Optional[bool] = Field(
```

Comment:
Shouldn't need this flag to preserve old behavior...  I don't think anyone will want the old behavior.